### DISCUSSION TITLE
Repeated failures building custom openwakeword on Google Colab for "computer"

### DISCUSSION THREAD

**Discussion by sovamind** [dscripka/openWakeWord#203] [General]
I'm running Home Assistant and having nothing but problems with the stock openwakeword words recognizing my deep American voice. I've been trying to build a custom wakeword using the published Google Collab but it kept timing out and failing and now I have zero credit hours left on a free account.

Is there anyone that has already generated an openwakeword for "computer" that could just share it for me to grab?

Alternatively, is there a way to generate the openwakeword locally?  Maybe a tutorial on how to install all the packages and run them in a Windows subsystem for linux session?

**Comment by dscripka** [comment]
That's a little odd, I also have a somewhat deep neutral-accent American english voice and in general performance seems fine. If you use openWakeWord by itself instead of via Home Assistant, is performance similar?

As for pre-trained models, there seems to be a large collection of community trained models here: https://github.com/fwartner/home-assistant-wakewords-collection

And yes, it is possible to run the training of a custom model locally, but it is not always straightforward to configure the environment correctly. I would start from this example notebook: https://github.com/dscripka/openWakeWord/blob/main/notebooks/automatic_model_training.ipynb

**Comment by sovamind** [comment]
Thanks for the link to the wakeword collection. I see "computer" in there so I can just install those files and I think it will meet my needs.
